### ISSUE TITLE
Default color option seems to have a "one off" issue.

### ISSUE THREAD

**Issue by MA-MacDonald** [tannerlinsley/react-charts#226]
Sorry I know you're probably sick of me...

For example: In order to apply colors properly to a two series chart you need to add three strings in the defaultColors array.

```javascript
      <Chart
        options={{
          defaultColors: ["#4c26c9", "#f44336"],          // doesnt work
          // defaultColors: ['#4c26c9', '#f44336', ""],   // works
          data: data,
          primaryAxis,
          secondaryAxes
        }}
      />
```

https://codesandbox.io/s/modern-flower-kdfy9?file=/src/App.js

**Comment by MA-MacDonald** [comment]
this is on beta.19

**Comment by tannerlinsley** [comment]
Should be fixed with e0e985cb8ac4c284dc369ed903ac4d065271e8c5 👍
